plutus-playground-client/src/MainFrame.purs

```diff
@@ -290,46 +299,85 @@ handleAction EvaluateActions =
         assign _evaluationResult Loading
         result <- lift $ postEvaluation evaluation
         assign _evaluationResult result
-        -- If we got a successful result, update last evaluated simulation and switch tab.
         case result of
-          Success (Left _) -> pure unit
-          _ -> do
-            updateSimulationOnSuccess result simulation
+          Success (Right _) -> do
+            -- on successful evaluation, update last evaluated simulation, and reset and show transactions
+            when (isSuccess result) do
+              (assign _lastEvaluatedSimulation simulation)
+              (assign _blockchainVisualisationState Chain.initialState)
             replaceViewOnSuccess result Simulations Transactions
+            lift $ scrollIntoView simulatorTitleRefLabel
+          Success (Left _) -> do
+            -- on failed evaluation, scroll the error pane into view
+            lift $ scrollIntoView simulationsErrorRefLabel
+          Failure _ -> do
+            -- on failed response, scroll the ajax error pane into view
+            lift $ scrollIntoView ajaxErrorRefLabel
```

Comment:
I think we are agreed that a `do` doesn't count as unused if we have a comment like this. ✔️